setup.py

```diff
@@ -75,7 +75,7 @@ def run(self):
 s3 = ["boto3==1.9.115"]
 azure = ["azure-storage-blob==2.1.0"]
 oss = ["oss2==2.6.1"]
-ssh = ["paramiko>=2.5.0"]
```

Comment:
This merge is a total mess, I can still see commits from other people in here. Is it how merges for PRs are or did you mess it up?

Reply:
solved